index.js

```diff
@@ -16,10 +17,13 @@ function AssetFingerprint(initializerDirectory, needsFingerprint = true) {
 AssetFingerprint.prototype.apply = function(compiler) {
   compiler.plugin('done', function(stats) {
     if (this.needsFingerprint) {
-      let output = `ASSET_FINGERPRINT = '${stats.hash}'`;
-      let initializerPath = `${this.initializerDirectory}/asset_fingerprint.rb`;
-
-      fs.writeFileSync(initializerPath, output);
+        let defaultRun = this.fingerprintName === 'ASSET_FINGERPRINT';
+        let initializerPath = `${this.initializerDirectory}/asset_fingerprint.rb`;
+        const newline = defaultRun ? '' : '\r\n';
+        const fsMethod = defaultRun ? 'writeFileSync' : 'appendFileSync';
+        let output = `${newline}${this.fingerprintName} = '${stats.hash}'`;
+        fs[fsMethod](initializerPath, output);
+        console.log(`asset-fingerprint-webpack-rails: updated file ${initializerPath} with ${this.fingerprintName} = ${stats.hash}`)
```

Comment:
This doesn't reflect any secondary append to the file, does it?

Reply:
It will continue to append unless default arguments, or the value for fingerprintName === 'ASSET_FINGERPRINT' are passed.

Reply:
Yeah, it should log with every run.